Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN    apt-get update                          \
 RUN    curl -sL https://deb.nodesource.com/setup_10.x | bash - \
     && apt-get update                                          \
     && apt-get upgrade --yes                                   \
-    && apt-get install --yes nodejs
+    && apt-get install --yes nodejs git
```

Comment:
This block is just for installing `nodejs`, please install it in the `apt-get` block above this one (line 7). Make sure to sort the list there too, and follow the same formatting.